charts/ibm-fhir-server/values.yaml

```
@@ -329,13 +331,37 @@ ingress:
     - secretName: ""
 resources:
   requests:
-    memory: "2Gi"
+    memory: "1Gi"
     ephemeral-storage: "1Gi"
   limits:
     memory: "5Gi"
     ephemeral-storage: "1Gi"
+# -- Pod affinity
+# @default -- Required node anti-affinity and preferred zone anti-affinity
+affinity: |
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchLabels:
+            {{- include "fhir.matchLabels" . | nindent 10 }}
+        topologyKey: kubernetes.io/hostname
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchLabels:
+              {{- include "fhir.matchLabels" . | nindent 12 }}
+          topologyKey: failure-domain.beta.kubernetes.io/zone
```

has been deprecated since v1.17 (https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesiozone) the successor is:

```suggestion
          topologyKey: topology.kubernetes.io/zone
```

---

thanks!